jni/src/knn_extension/faiss/MultiVectorResultCollectorFactory.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "MultiVectorResultCollectorFactory.h"
+#include "MultiVectorResultCollector.h"
+
+namespace os_faiss {
+
+MultiVectorResultCollectorFactory::MultiVectorResultCollectorFactory(BitSet* parent_bit_set)
+     : parent_bit_set(parent_bit_set) {}
+
+// id_map is set in IndexIDMap.cpp of faiss library with custom patch
+// https://github.com/heemin32/k-NN/blob/4c936f9e0c52a9303051c800037a981261d9c8a4/jni/patches/faiss/0001-Custom-patch-to-support-multi-vector.patch#L109
```

Comment:
link to opensearch repo?

Reply:
Thanks. Need to update. There was no such file in Opensearch repo as previous PR was not merged at the time of writing the code.